Minutes tech lead team meeting 5 March 2024 (#156)
ESMValGroup/Community: Tech Lead/Minutes/20240305.md

> ## Technical Lead Call 16 January 2024 (online; two weeks after design date)

Reviewer: ```suggestion
## Technical Lead Call 5 March 2024 (online; two weeks after design date)
```

    - Saskia Sloosveldt [@sloosvel](https://github.com/sloosvel)
    - Valeriu Predoi [@valeriupredoi](https://github.com/valeriupredoi)
    - Manuel Schlund [@schlunma](https://github.com/schlunma)
    - Emma Hogan [ehogan](https://github.com/ehogan)

## Agenda

- @ehogan: I have some questions about the release :blush:
- @schlunma: We at DLR consider writing a paper about computational improvements in ESMValTool (dask and co.)
- @schlunma: Allows specifying DRS per rootpath in configuration file (https://github.com/ESMValGroup/ESMValCore/pull/1894)
- @schlunma: Do we really need CMOR warnings? (https://github.com/ESMValGroup/ESMValCore/issues/2351)@schlunma: squash CMOR warnings
- @ehogan: The presentation I gave at RSECon23 is now available on [YouTube](https://www.youtube.com/watch?v=UVgvQXgWSRQ) :tada:

## Topics discussed

- @ehogan: release questions: one issue with the steps taken during the release - @bouweandela proposes a meeting to discuss the procedures (Bouwe is the replacement past release manager, so he's kind enough to help); Thursday 14th March 10am UK - @flicj191 Felicity and Rhaegar Zeng volunteered to look around;

> - @schlunma: paper on performance improvelemts with Dask, @bouweandela and @valeriupredoi are go for it

Reviewer: ```suggestion
- @schlunma: paper on performance improvements with Dask, @bouweandela and @valeriupredoi are go for it
```

> - @schlunma: specify DRS per rootpath: previous agreement at the June workshop was to include this in a larger-scale refactoring of the configuration - is this realistically feasable/possible, and when, and if not, we should merge https://github.com/ESMValGroup/ESMValCore/pull/1894; @bouweandela will try write up a synposis/set of specs for configuration overhaul - implementation looks distant, so we should merge the PR

Reviewer: ```suggestion
- @schlunma: specify DRS per rootpath: previous agreement at the June workshop was to include this in a larger-scale refactoring of the configuration - is this realistically feasable/possible, and when, and if not, we should merge https://github.com/ESMValGroup/ESMValCore/pull/1894; @bouweandela will write up a synposis/set of specs for configuration overhaul (see [ESMValGroup/ESMValCore#2371](https://github.com/ESMValGroup/ESMValCore/issues/2371)) - implementation looks distant, so we should merge the PR
```
